### PR TITLE
Fix for the issue: Github issue link is to be displayed on the task details page. #774

### PR DESCRIPTION
**Pull request by kapillondhe**
# What is the change?
Allow GitHub issue link from the client request. This will store the issue html URL in the task database. This link will be further used in website-status to open the git issue from the Task details page.

# Is Development Tested?

-  [x] Yes
-  [ ] No

# Note to Reviewers:
PR 815 raised in website-status for client related changes.

### CODE REVIEW

**Review by sahsisunny — CHANGES_REQUESTED** [review]
Where is the test?

**Comment by kapillondhe** [comment]
> Where is the test?

Added the tests. Thanks.

File: test/unit/middlewares/tasks-validator.test.js
Comment (vinit717) [inline test/unit/middlewares/tasks-validator.test.js:1]: please fix the test and the lint also 
And no need to to write unnecessary test

**Comment by kapillondhe** [comment]
PR #1428 Raised.

**Pull request closed without merging**